### ISSUE TITLE
Build for mac OSX

### ISSUE THREAD

**Issue by jacobzeng** [maxpert/marmot#50]
First of all, thanks for the cool tool.
Do you consider build the executable file for mac OSX (amd64 and arm64)?

**Comment by maxpert** [comment]
I believe you just need to pass GOOS and GOARCH environment variable while building binaries.

**Comment by maxpert** [comment]
Closing due to inactivity

**Comment by mindreframer** [comment]
@maxpert I got this bash script to compile the binary on MacBook AIR M1 (2020)

```bash
#!/bin/sh

CC=gcc-13 \
CXX=g++-13 \
GOOS=darwin GOARCH=arm64 CGO_ENABLED=1 \
go build -ldflags "-linkmode external" -o dist/osx/arm64/marmot
```

Maybe it helps someone. It would be great to also have binary releases for OSX, maybe with https://goreleaser.com/quick-start/ ? 

Cheers and thanks for the project, it looks great and the code is very clean!

**Comment by maxpert** [comment]
Given high demand I think it make sense to have binary builds for Mac in place as well

**Comment by maxpert** [comment]
#75 has been merged and `v0.8.5` already has binaries available. A follow up item might be to publish `brew` tap
